terratag_test.go

```diff
@@ -127,17 +127,17 @@ func itShouldGenerateExpectedTerratagFiles(suiteDir string, g *GomegaWithT) {
 }
 
 func itShouldRunTerraformValidate(entryDir string, g *GomegaWithT) {
-	err := terraform(entryDir, "validate")
+	err := run_terraform(entryDir, "validate")
```

Comment:
name conflicts...
modifed terraform function to run_terraform and terratag function to run_terratag.